src/utils/SdkCoin.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.25;
+
+// https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#IERC20Metadata
+import { IERC20Metadata } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import { IISdkCoinErrors } from "../errors/IISdkCoinErrors.sol";
+
+library SdkCoin {
```

Comment:
```suggestion
library SdkCoin is IISdkCoinErrors {
```

Reply:
Would be great but seems this is not possible for libraries! 
<img width="568" alt="Screenshot 2024-08-07 alle 12 24 43" src="https://github.com/user-attachments/assets/5655c769-a5c9-4797-ae3e-ac8de61ba641">